changelog/bitfields.dd

```diff
@@ -0,0 +1,3 @@
+Add bit fields to D
+
+They work just like the bit fields in ImportC do.
```

Comment:
I would add a link to a place where it is explained how ImportC/C bitfields work

Reply:
And add a code example

Reply:
I'll write the manual for it tomorrow. I'm done for the evening.

Reply:
added